src/advanced_inv.h

```diff
@@ -336,7 +351,13 @@ class advanced_inventory
 
         bool exit;
 
+        // store/load settings (such as index, filter, etc)
+        void save_settings(bool only_panes = false);
```

Comment:
The function is called at two places, an explicit value for the parameter is given at one of them. Why don't you add the default value to the other place (in the destructor) and make the parameter mandatory. That actually saves you 3 characters: ` = ` (-: